apps/bas/package.json

```diff
@@ -101,5 +101,11 @@
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-styled-components": "^0.1.1",
     "stylelint-processor-styled-components": "^1.10.0"
+  },
+  "vx": {
```

Comment:
I kinda like putting this in `package.json`

Reply:
The downside is that it doesn't work with non-nodejs apps/packages.